pkgs/development/libraries/vulkan-loader/system-search-path.patch

```diff
@@ -0,0 +1,45 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
```

Comment:
This isn't a requirement for merging, but it would be really nice to get this upstreamed. It seems like something that could get accepted.

Reply:
Yeah I will submit it, I have lots of things to do :)

Reply:
Submitted PR: https://github.com/KhronosGroup/Vulkan-Loader/pull/195